articles/javascript-vs-typescript/index.md

```diff
@@ -0,0 +1,242 @@
+[Typescript](https://www.typescriptlang.org/docs/) is a trendy programming language. It is a superset of JavaScript. Typescript can be described as an improved version of JavaScript. It inherits the functionalities and features of native JavaScript as well as adding more additional features and syntax. This means that JavaScript code is effectively Typescript code.
+
+On the other hand, JavaScript is an objected based scripting language used for creating interactive web pages. It is a well-established programming language. It is even the most popular programming language in the world of development, a survey conducted by [stack overflow](https://insights.stackoverflow.com/survey/2020#technology-programming-scripting-and-markup-languages-professional-developers)
+.
+
+![Most popular language stack overflow](/engineering-education/javascript-vs-typescript/most-popular-language-stack-overflow.jpg)
+
+[***Image Source***](https://insights.stackoverflow.com/survey/2020#technology-programming-scripting-and-markup-languages-professional-developers)
+
+The two languages have been remarkably used to develop superb web applications for server-side and client-side.
+
+### History: The beginning
+[Brendan Eich](https://en.wikipedia.org/wiki/Brendan_Eich), [Netscape](https://en.wikipedia.org/wiki/Netscape) Communications Corporation programmer, developed JavaScript. JavaScript was launched in [September 1995](https://en.wikipedia.org/wiki/JavaScript#History), initially called Mocha. It was created to ease web interactivity with Java-like syntax (object-oriented). JavaScript became a popular scripting tool and was later named JavaScript. This naming was adopted to reflect Java-like code in the Netscape's browser (Mozilla's ancestor). This scripting language becomes even more popular, and Netscape submitted it to [ECMA](https://www.ecma-international.org/) (European Computer Manufacturers Association) to attract more corporations. JavaScript now complies with [ECMAScript](https://en.wikipedia.org/wiki/ECMAScript) (ES) standard. [ES5](https://www.w3schools.com/js/js_es5.asp) and [ES6](https://www.w3schools.com/js/js_es6.asp) are the most used ES standards applied alongside JavaScript.
```

Comment:
Avoid too much reliance on Wikipedia since its not a credible or reliable source of information.